posthog/migrations/0185_special_migrations.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.5 on 2021-11-30 10:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0184_delete_sessionsfilter"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="SpecialMigration",
+            fields=[
+                ("id", models.BigAutoField(primary_key=True, serialize=False)),
+                ("name", models.CharField(max_length=50)),
+                ("description", models.CharField(blank=True, max_length=400, null=True)),
+                ("progress", models.IntegerField(default=0)),
```

Comment:
why isn't this `PositiveSmallIntegerField` too?

Reply:
good catch, updated